.pipelines/templates/.builder-release-template.yaml

```diff
@@ -300,4 +300,16 @@ steps:
       artifactName: '${{ parameters.artifactName }}'
       targetPath: '$(Build.ArtifactStagingDirectory)'
     displayName: Publish artifacts
-    condition: always()
\ No newline at end of file
+    condition: always()
+  - task: PublishPipelineArtifact@1
+    displayName: Publish publishing-info so e2e tests can be run locally
```

Comment:
what is this for? e2e's can already be run locally?

Reply:
oh, I think this was because of the bug to remove these, that should be fixed now with a merge from master

Reply:
Yup. Have removed this from the PR now.